doc/classes/ProjectSettings.xml

```diff
@@ -813,15 +813,18 @@
 		</member>
 		<member name="display/window/size/initial_position" type="Vector2i" setter="" getter="" default="Vector2i(0, 0)">
 			Main window initial position (in virtual desktop coordinates), this setting is used only if [member display/window/size/initial_position_type] is set to "Absolute" ([code]0[/code]).
+			[b]Note:[/b] This setting only affects the exported project (or project run from the command line), as running the project from the editor will make the project set its window position according to the [b]Run &gt; Window Placement[/b] editor settings instead.
```

Comment:
I'd rather reference the equivalent Editor Settings. You probably wanted to generalize it so that the user knows where to navigate, but that sounds like something the built-in docs should automatically format during generation...
I'd rather be explicit now.
```suggestion
			[b]Note:[/b] This setting only affects the exported project, or when the project is run from the command line. In the editor, the value of [member EditorSettings.run/window_placement/rect_custom_position] is used instead.
```